packages/eui/src/components/selectable/selectable_templates/selectable_template_sitewide_option.tsx

```diff
@@ -139,9 +159,20 @@ function renderOptionMeta(
         );
       }
 
+      const cssStyles = [
```

Comment:
Would a type guard make sense here to a) prevent direct casting and make it a bit more readable (that might be subjective though)? 🤔 

```suggestion
const isMetaType = (
  metaType: string
): metaType is keyof typeof styles.metaTypes => metaType in styles.metaTypes;
const cssStyles = [
  styles.euiSelectableTemplateSitewide__optionMeta,
  ...(meta.type && isMetaType(meta.type)
    ? [styles.metaTypes.fontWeight, styles.metaTypes[meta.type]]
    : [])
```

Reply:
I'm good either way! I'll add the inline `isMetaType` fn, that looks fun! 🎉

Reply:
https://github.com/elastic/eui/pull/7944/commits/1ffb335a36d0561325df71a000f8291cfdb8c24a